test/integration/targets/nxos_l3_interfaces/tasks/main.yaml

```diff
@@ -1,2 +1,13 @@
 ---
+- set_fact: nxos_int4="{{ intdataraw[4].interface }}"
```

Comment:
So, we are working to to bring nxos online for zuul, and don't have access to 4 interfaces. Can we refactor this to be nxos_int2 or nxos_int3?

Reply:
@chrisvanheuveln @mikewiebe Here's the failing run https://object-storage-ca-ymq-1.vexxhost.net/v1/a0b4156a37f9453eb4ec7db5422272df/ansible_42/65442/36707999cc13d951b048d574fa9c2e0c95eb1123/third-party-check/ansible-test-network-integration-nxos-cli-python36/a3e17de/controller/ara-report/result/f0a89e88-826a-4efe-8a03-0db65ee4622c/

Reply:
refactored to int3